core/src/main/java/com/amplifyframework/core/model/annotations/ModelConfig.java

```diff
@@ -40,4 +40,10 @@
      * @return the plural version of the name.
      */
     String pluralName() default "";
+
+    /**
+     * Specifies whether this model has owner based authorization.
```

Comment:
Would be beneficial is there is an example to demonstrate what this means OR add more description about it?

Reply:
Done